soccer/gameplay/tactics/wall.py

```diff
@@ -1,4 +1,6 @@
+from typing import Optional
 import composite_behavior
+from role_assignment import RoleRequirements
```

Comment:
`from ... import ..` should probably be `import ...`

Reply:
done